docs/man_pages/project/testing/publish.md

```diff
@@ -0,0 +1,44 @@
+publish
```

Comment:
Currently this is just a hollow command - that is to say, it's not implemented in any way - only a help file is present in case any user types in `$ tns publish -h`, given that `$ tns publish ios` is fully operational